src/internal/connector/onedrive/collections_test.go

```diff
@@ -724,6 +724,34 @@ func (suite *OneDriveCollectionsSuite) TestUpdateCollections() {
 			},
 			expectedExcludes: map[string]struct{}{},
 		},
+		{
```

Comment:
you could probably leave a test in one or the other test functions. They overlap a lot and we should probably merge `TestUpdateCollections` with `TestGet` at some point since the latter is a superset of functionality of the former 😅